app/controllers/optOut/ConfirmedOptOutController.scala

```diff
@@ -50,12 +53,18 @@ class ConfirmedOptOutController @Inject()(val authenticate: AuthenticationPredic
 
   def show(): Action[AnyContent] = auth.authenticatedAction(isAgent = false) {
     implicit user =>
-      Future.successful(Ok(confirmedOptOut(false)))
+      optOutService.makeOptOutUpdateRequest().map {
```

Comment:
should fix: Looks like view and update request is handled with one call. The proper way in my opinion is to create a submit api with redirects to view api

Reply:
please provide an example